decidim-core/app/functions/decidim/core/participatory_space_finder_base.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # A very basic resolver for the GraphQL endpoint for a single participatory spaces
+    # This can be easily overwritten by the participatory_space_manifest.query_finder
```

Comment:
Do you mean that this is the default resolver for all spaces?

Reply:
The default is  [participatory_space_finder.rb](https://github.com/Platoniq/decidim/blob/2bb6661be635d2df07908bf3190d66e600ca4150/decidim-core/app/functions/decidim/core/participatory_space_finder.rb) with inherits from this one (see [participatory_space_manifest.rb](https://github.com/Platoniq/decidim/blob/2bb6661be635d2df07908bf3190d66e600ca4150/decidim-core/lib/decidim/participatory_space_manifest.rb#L27)). It behaves as the former situation, defining and ID and nothing else. This is just a convenient base to create finder by just adding the `argument` method. Now is easy to add any other method to refer to that participatory space (using a `slug` for instance).

Example:
[participatory_process_finder.rb](https://github.com/Platoniq/decidim/blob/2bb6661be635d2df07908bf3190d66e600ca4150/decidim-participatory_processes/app/functions/decidim/participatory_processes/participatory_process_finder.rb#L7)

Similar approach is been taken in components and for listing (finder is for one result only)